portal/urls.py

```diff
@@ -16,9 +16,19 @@
 from django.contrib import admin
 from django.urls import path, include
 from . import views
- 
+# <<<<<<< harshborse
+from checkin import views as checkinViews
+
+# urlpatterns = [
+#     path('admin/', admin.site.urls),
+#     path('', views.home),
+#     path('checkin/', views.checkin),
+#     path('checkin/checkinGen', checkinViews.checkinGen),
+# =======
+
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('home/', views.home),
     path('checkin/', include('checkin.urls'))
+# >>>>>>> master
```

Comment:
Merge conflicts, not resolved properly

Reply:
i don't know the problem here sir,
it seems working